tests/selenium_tests.py

```diff
@@ -0,0 +1,135 @@
+        name.send_keys(location_name)
+        type_outdoor = driver.find_element_by_name('type').find_element_by_xpath('//option[@value="outdoor"]')
+        type_outdoor.click()
+        is_mobile = driver.find_element_by_name('is_mobile')
+        is_mobile.click()
+        save_button = driver.find_element_by_name('_save')
+        save_button.click()
+
+    def get_locations(self, driver, location_name):
+        # return all locations with given name
+        driver.get('http://dashboard.openwisp.org/admin/geo/location/')
+        locations = driver.find_elements_by_class_name('field-name')
+        location_name_locations = []
+        for location in locations:
+            try:
+                if location.find_element_by_link_text(location_name):
+                    location_name_locations.append(location)
+            except NoSuchElementException:
+                pass
+        return location_name_locations
+
+    def get_location_change_view(self, driver, location_name):
+        # redirects driver to change view of given location
+        location_name_locations = self.get_locations(driver, location_name)
+        if len(location_name_locations) == 0:
+            message = 'There is no location with {} name'.format(location_name)
+            self.fail(message)
+        elif len(location_name_locations) >= 2:
+            message = 'Can\'t find right object if there is more than one locactions with the same name'
+            self.fail(message)
+        test_location = driver.find_element_by_link_text(location_name)
+        test_location.click()
+
+    def set_location_marker(self, driver, location_name):
+        # set marker on map on location object
+        self.get_location_change_view(driver, location_name)
+        marker_button = driver.find_element_by_class_name('leaflet-draw-draw-marker')
+        marker_button.click()
+        geo_map = driver.find_element_by_id('id_geometry-map')
+        geo_map.click()
+        save_button = driver.find_element_by_name('_save')
+        save_button.click()
+
+    def delete_location(self, driver, location_name):
+        # delete all locations with location_name name
+        locations_to_delete = self.get_locations(driver, location_name)
+        for i in range(len(locations_to_delete)):
+            location = self.get_locations(driver, location_name)[0]
+            location.find_element_by_link_text(location_name).click()
+            delete_button = (
+                driver.find_element_by_class_name('submit-row').
+                find_element_by_class_name('deletelink-box')
+            )
+            delete_button.click()
+            confirm_delete_button = (
+                driver.
+                find_element_by_xpath('//div/div[@id="content"]/form/div/input[@type="submit"]')
+            )
+            confirm_delete_button.click()
+
+
+class TestWebSocket(TestUtilities, unittest.TestCase):
+    base_driver = webdriver.Chrome()
```

Comment:
Can we please have an option to have Chrome headless?
Like if an arguement `--headless` is passed to python scripts, it starts all the chrome instances as  headless.
Also, please enable headless on `.travis.yml`